surfer/viz.py

```diff
@@ -1833,30 +1854,59 @@ def set_surf(self, surf):
                 brain._f.scene.reset_zoom()
 
     @verbose
-    def scale_data_colormap(self, fmin, fmid, fmax, transparent, verbose=None):
+    def scale_data_colormap(self, fmin, fmid, fmax, transparent, 
+                            center=None, alpha=1.0, verbose=None):
         """Scale the data colormap.
+        
+        The colormap may be sequential or divergent. When the colormap is 
+        divergent indicate this by providing a value for 'center'. The 
+        meanings of fmin, fmid and fmax are different for sequential and 
+        divergent colormaps. For sequential colormaps the colormap is 
+        characterised by:
+            
+            [fmin, fmid, fmax]
+        
+        where fmin and fmax define the edges of the colormap and fmid will be
+        the value mapped to the center of the originally chosen colormap. For
+        divergent colormaps the colormap is characterised by
+            
+            [center-fmax, center-fmid, center-fmin, center, 
+             center+fmin, center+fmid, center+fmax]
+        
+        i.e., values between center-fmin and center+fmin will not be shown 
+        while center-fmid will map to the middle of the first half of the 
+        original colormap and center-fmid to the middle of the second half.
 
         Parameters
         ----------
         fmin : float
-            minimum value of colormap
+            minimum value for colormap
         fmid : float
             value corresponding to color midpoint
         fmax : float
             maximum value for colormap
         transparent : boolean
-            if True: use a linear transparency between fmin and fmid
+            if True: use a linear transparency between fmin and fmid, or 
+            between center-fmin to center-fmid and center+fmin to center+fmid
```

Comment:
How about "... linear transparency between fmin and fmid (symmetrically for divergent colormaps)."